missing_keys.txt

```diff
@@ -47,7 +47,7 @@
 |  all   | spree.remove                                                                 | app/views/spree/admin/taxons/_taxon_table.html.haml:15                                             | - missing translation, added key -> not sure though, where to trigger this missing translation
 |  all   | spree.shipment_mailer.base_subject.picked_up_subject                         | app/mailers/spree/shipment_mailer.rb:15                                                            | - exists in en.yml, seems to be a false positive
 |  all   | spree.taxon_edit                                                             | app/views/spree/admin/taxons/edit.html.haml:4                                                      | - missing translation, added key
-|  all   | thank_you_for_your_order                                                     | app/views/spree/orders/show.html.haml:26                                                           |
+|  all   | thank_you_for_your_order                                                     | app/views/spree/orders/show.html.haml:26                                                           | - missing translation, added key
 |  all   | unrecognized_card_type                                                       | app/views/spree/admin/payments/source_forms/_gateway.html.haml:25                                  |
```

Comment:
Another option could have been to reuse the key `thanks: "Thank you for your business."`.